tests/common/tests/cca_test.hpp

```diff
@@ -112,15 +114,16 @@ class ConnectedComponentAnalysisTests
 
         cca_truth_hit_reader truth_reader(file_truth);
 
+        traccc::scalar var_adjustment = (pitch * pitch) / 12.;
+
         cca_truth_hit io_truth;
         while (truth_reader.read(io_truth)) {
             ASSERT_TRUE(result.find(io_truth.geometry_id) != result.end());
 
             const vecmem::vector<traccc::measurement> &meas =
                 result.at(io_truth.geometry_id);
 
-            traccc::scalar tol = std::max(
-                0.1, 0.0001 * std::max(io_truth.channel0, io_truth.channel1));
+            traccc::scalar tol = 0.0001;
```

Comment:
:thinking: This is just a subjective comment, but you could make all the new variables `const`. Plus you should probably use `f` on all the literal constants.

Reply:
Fair point, that will save poor @fredevb some work later. :slightly_smiling_face: